config/config.go

```diff
@@ -58,6 +58,8 @@ type Config struct {
 	SourceBlockchains      []*SourceBlockchain      `mapstructure:"source-blockchains" json:"source-blockchains"`
 	DestinationBlockchains []*DestinationBlockchain `mapstructure:"destination-blockchains" json:"destination-blockchains"`
 	ProcessMissedBlocks    bool                     `mapstructure:"process-missed-blocks" json:"process-missed-blocks"`
+	DeciderHost            string                   `mapstructure:"decider-host" json:"decider-host"`
+	DeciderPort            *uint16                  `mapstructure:"decider-port" json:"decider-port"`
```

Comment:
The way I have it coded right now, you can skip specifying a host, and it will assume `localhost`, which felt nice for the case when the decider is running locally. It's nice, but not a huge deal. I'm definitely open to other opinions.